src/drawing/conics.rs

```diff
@@ -310,8 +310,15 @@ where
 
 #[cfg(test)]
 mod tests {
+    use super::draw_filled_ellipse_mut;
     use image::{GrayImage, Luma};
 
+    #[test]
+    fn draw_huge_ellipse() {
```

Comment:
Without an explanation this test with no assertions is pretty unclear.

This could be improved with a comment or by changing the test name, but ideally you could add assertions that some points along the ellipse are drawn - maybe the horizontal and vertical min/max points?

Reply:
It's just a panic if an old version of a function is used.

Reply:
I meant that it wouldn’t be clear why this test existed for someone reading it in future.

Reply:
Ok, I wrote a validation for a filled ellipse